### DISCUSSION TITLE
Light/Dark Mode Toggle

### DISCUSSION THREAD

**Discussion by hcbottger** [notaapp/talk#258] [Feature Requests]
Hi team, 
Loving my early impressions of Nota so far <3

Are you planning to add a toggle between light and dark mode? My MacOS is dark-theme, but there are a few things I prefer light (such as my note taking app!). Hoping to decouple from my OS theme.

Thanks!

**Comment by astoilkov** [comment]
Hi. Great to hear that ❤️.

Yes, it makes sense. Other users have requested this as well. We will probably do it.

**Comment by alexanderheckel** [comment]
The lack of this feature is the reason why I personally don't use this app. Therefore, I would also like to have this feature ;)

**Comment by vladstudio** [comment]
+1 for the feature. It is more or less common to offer dark/light/auto mode for Mac apps (where auto means synced with OS). Discord goes 1 step further and offers always-dark sidebar - which I personally prefer!

<img width="706" alt="image" src="https://user-images.githubusercontent.com/914320/180191855-bdd88939-5c64-43fb-a1d2-0e8cec911850.png">

**Comment by hcbottger** [comment]
Hey team, just curious on an eta for this? Noticed the in-app prompt for buying a license -- happy to do so with confirmation this is coming soon. thanks so much

**Reply by hcbottger** [reply]
Nevermind! Just saw the update -- this is awesome 😎. Dropping my credit card in now, keep up the great work guys!

**Comment by astoilkov** [comment]
The manual dark/light theme toggle is now part of the 0.38 release. You can find it in the command palette by typing "theme".

Thanks for all the feedback. It fast-forwarded the implementation of this feature.

**Reply by dmitrimaximov** [reply]
thank you for posting this. I looked through all the menus and settings and couldn't find it. the thought to use command palette didn't come across my mind. you just made my day!
